extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -504,6 +504,10 @@ private function saveUsersAvatar( $userId = null, $data = null ) {
 					if ( empty( $wgAvatarsUseService ) ) {
 						$user->setGlobalAttribute( AVATAR_USER_OPTION_NAME, $data->file );
 					}
+					else {
```

Comment:
Since Wikia's coding conventions are a superset of MediaWiki's coding conventions, we should follow the rule that [an else is placed on the same line as the previous closing brace](https://www.mediawiki.org/wiki/Manual:Coding_conventions#Indenting_and_alignment).